src/Workspaces/Core/Portable/Workspace/Solution/RecoverableTextAndVersion.RecoverableText.cs

```diff
@@ -100,7 +100,7 @@ public override SourceText GetValue(CancellationToken cancellationToken)
             }
         }
 
-        public override async Task<SourceText> GetValueAsync(CancellationToken cancellationToken)
+        public async Task<SourceText> GetValueAsync(CancellationToken cancellationToken)
```

Comment:
all three of these methods are used by the containing type.  so we do have to keep all this functionality for now.